static/pattern-library/sass/main-ltr.scss

```diff
@@ -14,7 +14,7 @@
 // ------------------------------
 // #UTILITIES
 // ------------------------------
-@import 'utilities/variables';
+@import 'utilities/variables-ltr';
```

Comment:
NOTE: the non LTR/RTL dependent ``_variables.scss`` partial is included in the shared ``_build.scss`` partial using in both LTR/RTL compile files - https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static/pattern-library/sass/_build.scss